@xen-orchestra/fs/src/s3.js

```diff
@@ -1,12 +1,26 @@
-import aws from '@sullux/aws-sdk'
+import {
+  AbortMultipartUploadCommand,
+  CompleteMultipartUploadCommand,
+  CopyObjectCommand,
+  CreateMultipartUploadCommand,
+  DeleteObjectCommand,
+  GetObjectCommand,
+  HeadObjectCommand,
+  ListObjectsV2Command,
+  PutObjectCommand,
+  S3Client, UploadPartCommand,
+  UploadPartCopyCommand,
+} from '@aws-sdk/client-s3';
+import { Upload } from '@aws-sdk/lib-storage';
+import { NodeHttpHandler } from '@aws-sdk/node-http-handler';
 import assert from 'assert'
-import http from 'http'
-import https from 'https'
+import { Agent as HttpAgent } from 'http'
+import { Agent as HttpsAgent } from 'https';
 import pRetry from 'promise-toolbox/retry'
 import { createLogger } from '@xen-orchestra/log'
 import { decorateWith } from '@vates/decorate-with'
+import { PassThrough, pipeline } from 'stream';
```

Comment:
You're adding semicolons, I feel like Prettier does not run as it should when you're creating a commit.

Either you are bypassing hooks (`git commit -n`) or something is not working in your setup, let me know if you need help.

Reply:
I wouldn't have allowed myself to force a commit.

I'll check my setup.